cli/stork-publisher-agent/runner.go

```diff
@@ -76,10 +76,12 @@ func (r *PublisherAgentRunner[T]) UpdateBrokerConnections() {
 
 	// add or update desired connections
 	for brokerUrl, newAssetIdMap := range newBrokerMap {
-		_, exists := r.brokerMap[brokerUrl]
-		if exists {
+		_, brokerExists := r.brokerMap[brokerUrl]
+		outgoingConnection, outgoingConnectionExists := r.outgoingConnections[brokerUrl]
```

Comment:
got a nil reference error when I was trying to break things and refreshing the registry really frequently - if an outgoing connection object is missing we should create it.